scripts/download_model_from_gist.sh

```diff
@@ -18,7 +18,7 @@ fi
 
 echo "Downloading Caffe model info to $MODEL_DIR ..."
 mkdir -p $MODEL_DIR
-wget https://gist.github.com/$GIST/download -O $MODEL_DIR/gist.tar.gz
-tar xzf $MODEL_DIR/gist.tar.gz --directory=$MODEL_DIR --strip-components=1
-rm $MODEL_DIR/gist.tar.gz
+wget https://gist.github.com/$GIST/download -O $MODEL_DIR/gist.zip
```

Comment:
when `https://gist.github.com/$GIST/download` is downloaded, you have a zip, not a tarball, hence the need for this patch.
If there is another URL to grab a tarball, then I agree that it'd be better to fix the URL and stick with the old tarball extraction.